ajax/libs/jReject/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "jReject",
+  "description": "jReject is a simple tool for rejecting specific browsers in a nice modal window",
+  "filename": "js/jquery.reject.min.js",
+  "keywords": [
+    "browser",
+    "reject",
+    "popup"
+  ],
+  "version": "1.1.4",
+  "author": {
+    "name": "Steven Bower",
+    "url": "https://github.com/BluSyn"
+  },
+  "licenses": [
+    "MIT",
+    "GPLv2"
```

Comment:
According to #5543, we couldn't recognize "GPLv2" but "GPL-2.0". (see [SPDX License List](https://spdx.org/licenses/))
Please use `"GPL-2.0"`.